setup.py

```diff
@@ -51,6 +51,7 @@
     install_requires=[
         "matplotlib>=2.1.1",
         "networkx>=2.5.1",
+        "pandas",
```

Comment:
We want to avoid including extra heavy dependencies as much as possible, and so far got away without `pandas` 😅  Wonder if it would be easy to use `csv.DictReader` here and operate on dicts?